chalicelib/action.py

```diff
@@ -357,14 +362,29 @@ def perform_action(self):
                 message=f"Delete doesn't support date range :cry:"
             )
 
-        if not self._check_locks(date=date["from"], second_date=date["to"]):
-            self.send_attachment(
-                attachment=delete_message_menu(self.user_name, date_string)
+        if self._check_locks(date=date["from"], second_date=date["to"]):
+            return self.send_response(
+                message=f"Unable to delete since month is locked :cry:"
+            )
+
+        if not self._has_events(date["from"]):
```

Comment:
Use the existing date_string instead
```suggestion
        if not self._has_events(date=date_string):
```

The other suggestions is depending on this change

Reply:
Doesn't this break `delete today`? `date_string` is the data as written in slack and it seems the api doesn't support `today`. Even if we're okay with removing `today` I prefer to use the cleaned data before passing it (harder to pass invalid data trough)

Reply:
Good point